docs-new/assets/javascripts/versions.js

```diff
@@ -1,5 +1,6 @@
 // VERSION WARNINGS
 window.addEventListener("DOMContentLoaded", function() {
+    return; //TODO: decide if we want this
```

Comment:
I think we should discuss if we want this banner and, in case, disable for local builds

Reply:
i would keep it. it makes sense to have something like this and the only other option for such an announcement banner is to have it implemented statically which is not really what we want I think.